lighthouse-core/test/gather/computed/network-throughput-test.js

```diff
@@ -26,8 +26,6 @@ describe('NetworkThroughput', () => {
   }
 
   it('should return Infinity for no/missing records', () => {
-    assert.equal(compute(undefined), Infinity);
-    assert.equal(compute(null), Infinity);
```

Comment:
since the network records will now always come from `requestNetworkRecords`, there will always be *some* array, so these aren't necessary anymore. I may be missing a failure mode, though, because I don't remember how these were possible in the first place. Please clue me in if I am missing something :)